### DISCUSSION TITLE
translation/generate-and-store-opentelemetry-traces-automatically/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#21] [Announcements]
# translation/generate-and-store-opentelemetry-traces-automatically/

首先，我们将解释一下如何在 Kubernetes 自动生成和存储 OpenTelemetry 追踪，剖析 OpenTelemetry Operator 在内部的真正作用。接下来，我们将通过一个例子演示如何将其直接付诸实践。

https://lib.jimmysong.io/translation/generate-and-store-opentelemetry-traces-automatically/

**Comment by rootsongjc** [comment]
这篇文章让你使用 Helm 安装开源可观察性项目 [Tobs](https://github.com/timescale/tobs)，它内置了 Prometheus、Grafana 还有 metric 收集，将数据发送给 OpenTelemetry Collector。所谓自动化就是可以使用 Helm 安装，OpenTelemetry Operator。
